setup.cfg

```diff
@@ -11,4 +11,4 @@ exclude =
 
 [tool:pytest]
 addopts = --flake8
-flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
+flake8-ignore = E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503 E203
```

Comment:
Same here. You can move this to pyproject.toml if you like (you can't move the flake8 block):

https://github.com/lsst/daf_butler/blob/main/pyproject.toml#L119-L121